core/tests/org.openjdk.jmc.common.test/src/main/java/org/openjdk/jmc/common/test/util/MCStackTraceTest.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hi Richard! Can you please update the copyright to also include the Datadog copyright and the correct year.